src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -230,48 +230,48 @@ LedgerConsensusImp::LedgerConsensusImp (
         LocalTxs& localtx,
         LedgerMaster& ledgerMaster,
         FeeVote& feeVote)
-    : app_ (app)
```

Comment:
That's unfortunate. Leading underscores look so much better. Look how easy it is to see what this code does: https://github.com/ripple/rippled/blob/06e3ac5e2cff32857e082d332d227278f7637a82/src/ripple/app/ledger/impl/LedgerConsensusImp.cpp#L1676-L1701